### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by idfunctor** [wojtekmaj/react-daterange-picker#237]
This component is not keyboard accessible at all. I can't use arrow keys to navigate between dates. It is impossible to navigate dates using tab character only.
Is there a plan to make the component accessible?

**Comment by wojtekmaj** [comment]
I believe you're talking about React-Calendar. Keyboard navigation improvements are definitely on the roadmap, tracked separately, but there is no timeline due to insufficient funding.
